### ISSUE TITLE
warning ---  exported type should have comment or be unexported

### ISSUE THREAD

**Issue by jcmartins** [joefitzgerald/gometalinter-linter#34]
Why I have this warning? 
Can I disable this warning?

exported type Person should have comment or be unexported (golint)at line 11 col 6

The line 11 is:
type Person struct {
    Name  string
    Phone string
}

**Comment by joefitzgerald** [comment]
https://github.com/joefitzgerald/gometalinter-linter#when-viewed-in-the-package-settings-dialog

Just remove the --enable=golint item, if you don't want golint warnings.

**Comment by TIGERB** [comment]
Why the golint gave this warning? The true way is what? thx~

**Comment by lloiser** [comment]
Just add a meaningful comment to the struct
```go
// Person is a representation of a person
type Person struct {
    Name string
    Phone string
}
```

**Comment by TIGERB** [comment]
@lloiser thx~

**Comment by SebastianLavertheDe** [comment]
@lloiser , thx 。 By the way, the first word of the comment must be the same as the struct name

**Comment by TIGERB** [comment]
@leoliuzcl Just public struct or method
